tests/Integration/Validation/DonationValidatorTest.php

```diff
@@ -207,4 +212,10 @@ private function newBankDataValidator() {
 		return new BankDataValidator( $ibanValidatorMock );
 	}
 
+	private function newMockPersonalInfoValidator() {
```

Comment:
https://github.com/wmde/FundraisingFrontend/pull/245

Esp nice when using mocks, since else PhpStorm gives you a type warning